email_alert_notifications/jenkins.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+mkdir -p archive/$(date +%s)
+mv *.zip archive/$(date +$s)/
+/usr/bin/zip rename_email_files_with_request_id.py.zip rename_email_files_with_request_id.py || exit 1
```

Comment:
What's the reason for the `|| exit 1` at the end of the line? Won't the `set -e` catch that?